e2e/utils/touch.js

```diff
@@ -48,7 +48,7 @@ const dragCardTo = async(browser, cardSelector, to)=>{
 		return prev.then(()=>{
 			executeTouch(browser, cardSelector, value, 'touchmove');
 
-			return new Promise(resolve => setTimeout(resolve, 10));
+			return new Promise(resolve => setTimeout(resolve, 100));
```

Comment:
Without this (maybe the number can be lower), dragging the card was too fast so the card did not stay minimized.